src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/glide/GlideLogging.java

```diff
@@ -1,18 +0,0 @@
-package com.microsoft.maui.glide;
-
-import android.util.Log;
-
-public class GlideLogging {
-    private static final String TAG = "Glide";
-    private static final boolean IS_VERBOSE_LOGGABLE = Log.isLoggable(TAG, Log.VERBOSE);
-
-    public static boolean isVerboseLoggable() {
-        return IS_VERBOSE_LOGGABLE;
-    }
-
-    public static void v(String message) {
-        if (IS_VERBOSE_LOGGABLE) {
-            Log.v(TAG, message);
-        }
-    }
-}
```

Comment:
It is not good because then we would log everything as "Glide", while we may want to be specific to the context where the context is created.

I wanted to follow the `ILogger<T>` approach where every class is responsible for their own logs.

If I just need to debug the CustomViewTarget class I don't want to spam my logs with all of the glide logs.

Reply:
Could this instead be `JavaLogging.cs`, a non-static class, and you pass in the `TAG` in the constructor?

Reply:
I've implemented the requested change:
```
private static final PlatformLogger logger = new PlatformLogger("FontModelResourceDecoder");
// ...
if (logger.isVerboseLoggable) logger.v(model.toString());
```